Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/sites/Wired7.java

```diff
@@ -82,7 +82,6 @@ public void setup() {
         setIcon(SiteIcon.fromFavicon(HttpUrl.parse("https://wired-7.org/favicon.ico")));
 
         setBoards(
-                Board.fromSiteNameCode(this, "Lewds & +18", "18"),
```

Comment:
If you're deleting a board, you'll need to update DatabaseHelper and use the deleteBoard method. See https://github.com/Adamantcheese/Kuroba/commit/ebdae105581cf727ef1a679ee33c3b87fae5a4c9.

Reply:
Got it, thanks.